packages/keyvAdapter/src/index.ts

```diff
@@ -39,11 +40,7 @@ export class KeyvAdapter<
     return this.dataLoader ? this.dataLoader.load(key) : this.keyv.get(key);
   }
 
-  async set(
-    key: string,
-    value: V,
-    opts?: KeyValueCacheSetOptions,
-  ): Promise<void> {
+  async set(key: string, value: V, opts?: SO): Promise<void> {
```

Comment:
I think this change is similar, but different.

Here the underlying cache is explicitly a `Keyv` (rather than some implementation of the `KeyValueCache` interface, so defining the `SO` here doesn't make sense to me (since we can't pass these arbitrary options to `Keyv` in this case). So I think in this case the type change is misleading and we can't resolve that with a code change.

Reply:
@trevor-scheer Yeah, agreed - I'll revert.

I guess if we want extended set options with a Redis cache, we'd just need to implement a custom Redis KeyValueCache instead of using the KeyvAdapter + KeyvRedis

Reply:
@trevor-scheer I've reverted the change.